packages/html/src/checkbox/checkbox-group.spec.tsx

```diff
@@ -0,0 +1,47 @@
+import { classNames } from '../misc';
+
+export const CHECKBOXGROUP_CLASSNAME = `k-checkbox-list`;
+
+const states = [];
+
+const options = {};
+
+export type KendoCheckboxGroupProps = {
+    layout?: 'vertical' | 'horizontal';
+};
+
+const defaultProps = {
+    layout: 'vertical',
+};
+
+export const CheckboxGroup = (
+    props: KendoCheckboxGroupProps &
+        React.HTMLAttributes<HTMLUListElement>
+) => {
+    const {
+        layout = defaultProps.layout,
+        ...other
+    } = props;
+
+    return (
+        <ul
+            {...other}
+            className={classNames(
+                props.className,
+                CHECKBOXGROUP_CLASSNAME,
+                {
+                    [`k-list-${layout}`]: layout === 'horizontal',
```

Comment:
I think we should render **_k-list-vertical_** class by default, as well as when _layout_ is set to _**vertical**_